packages/swing-store-lmdb/src/snapStore.js

```diff
@@ -130,5 +138,22 @@ export function makeSnapStore(
     }, `${hash}-load`);
   }
 
-  return freeze({ load, save });
+  /**
+   * @param {string} hash
+   */
+  function prepareToDelete(hash) {
+    toDelete.add(hash);
+  }
+
+  /**
+   * @throws { Error } if any call to unlink() fails
+   */
+  function commitDeletes() {
+    for (const hash of toDelete) {
```

Comment:
We need to clear `toDelete`, yeah? And maybe have a test that calls `commitDeletes` twice to provde it?

Reply:
oh my. that was sloppy.